vision/automl/edge_container_predict/automl_vision_edge_container_predict_test.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for automl_vision_edge_container_predict.
+
+The test will automatically start a container with a sample saved_model.pb, send
+a request with one image, verify the response and delete the started container.
+
+If you want to try the test, please install
+[gsutil tools](https://cloud.google.com/storage/docs/gsutil_install) and
+[Docker CE](https://docs.docker.com/install/) first.
+
+Examples:
+sudo python automl_vision_edge_container_predict_test.py
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import subprocess
+import time
+import unittest
```

Comment:
conventionally we have been using `pytest` instead of `unittest` in this repo.  @beccasaurus please weigh in on this.

Reply:
done. rewrite the test in pytest.